languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -115936,6 +115936,1297 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             <example correction="prie">Je vous en <marker>pris</marker>.</example>
             <example correction="prie">Je t'en <marker>pris</marker>.</example>
         </rule>
+        <rulegroup id="VERBE_SUIVI_D_UN_NOM" name="avoir faim">
+            <!-- Source: https://fr.wiktionary.org/w/index.php?title=Cat%C3%A9gorie:Verbes_en_fran%C3%A7ais_suivis_d%E2%80%99un_nom_commun_sans_d%C3%A9terminant&pageuntil=faire+pression#mw-pages -->
+            <rule id="ATTENDRE_PLUS_NOM_INCORRECT" name="attendre suivi d'un nom incorrect" default="temp_off">
+                <pattern>
+                    <token postag="V.*" postag_regexp="yes" inflected="yes">attendre</token>
+                    <marker>
+                        <token postag="N.*" postag_regexp="yes">
+                            <exception>famille</exception>
+                            <exception>godot</exception>
+                        </token>
+                    </marker>
+                </pattern>
+                <message>Après le verbe 'attendre', ce nom n'est généralement pas utilisé.</message>
+                <suggestion>un \2</suggestion>
+                <suggestion>une \2</suggestion>
+                <suggestion>des \2</suggestion>
+                <example correction="un voiture|une voiture|des voiture">J'attends <marker>voiture</marker>.</example>
+                <example>J'attends <marker>famille</marker>.</example>
+            </rule>
+            <rule id="ATTRAPER_PLUS_NOM_INCORRECT" name="attraper suivi d'un nom incorrect" default="temp_off">
+                <pattern>
+                    <token postag="V.*" postag_regexp="yes" inflected="yes">attraper</token>
+                    <marker>
+                        <token postag="N.*" postag_regexp="yes">
+                            <exception>froid</exception>
+                        </token>
+                    </marker>
+                </pattern>
+                <message>Après le verbe 'attraper', ce nom n'est généralement pas utilisé.</message>
+                <suggestion>un \2</suggestion>
+                <suggestion>une \2</suggestion>
+                <suggestion>des \2</suggestion>
+                <example correction="un rhume|une rhume|des rhume">J'attrape <marker>rhume</marker>.</example>
+                <example>J'attrape <marker>froid</marker>.</example>
+            </rule>
+            <rule id="AVOIR_PLUS_NOM_INCORRECT" name="avoir suivi d'un nom incorrect" default="temp_off">
+                <pattern>
+                    <token postag="V.*" postag_regexp="yes" inflected="yes">avoir</token>
+                    <marker>
+                        <token postag="N.*" postag_regexp="yes">
+                            <exception>accès</exception>
```

Comment:
**suggestion(format):** here and everywhere in the PR, change the format of the list to a shorter, more readable and manageable list, like this line (there are many others in the file):
`<exception scope="next" inflected="yes" regexp="yes">humain|et|ou|de</exception></token>`

Reply:
I don't think that "inflected" can combine with "regexp". At least, chatGPT says it doesn't. I can replace this in most of the rules except the last one.

Reply:
Testing any rule from the grammar.xml file that contains the sequence can prove ChatGPT isn't always right 😉 
See for example the matches found for this rule (randomly selected amongst the 678 matches for "inflected="yes" regexp="yes"" in the xml file)

```
            <rule>
                <pattern>
                    <token inflected="yes" regexp="yes">accroupir|assoir|étendre|cacher|chier|coucher|cracher|dormir|jeter|poser|rouler|tomber|traîner|transport|vomir</token>
                    <marker>
                        <token>parterre</token>
                    </marker>
                </pattern>
                <message>Confusion entre le nom « parterre » et la locution « par terre » (sur le sol).</message>
                <suggestion>par terre</suggestion>
                <example correction="par terre">Ils dorment <marker>parterre</marker>.</example>
            </rule>
```

will have matches like `Les dames assises parterre lisent ou brodent en compatissant, Les chorégraphies sur cette chansons sont telles qu'avant le refrain, toute la foule s'accroupit parterre.`
It was, though, just an example. You might not even want to inflect the tokens in the list.